spec/fortnox/api/mappers/unit_spec.rb

```diff
@@ -0,0 +1,55 @@
+require 'spec_helper'
+require 'fortnox/api'
+require 'fortnox/api/mappers/unit'
+require 'fortnox/api/models/unit'
+require 'fortnox/api/mappers/examples/mapper'
+
+module Fortnox
```

Comment:
Why the modules? The usual way to do this is to write `describe Fortnox::API::Mapper::Unit`.